backend/cmm_helpers.ml

```diff
@@ -2247,6 +2247,15 @@ let arraylength kind arg dbg =
    to Arbitrary_effects and Has_coeffects, resp.
    Check if this can be improved (e.g., bswap). *)
 
+let if_operation_supported op ~f =
```

Comment:
(Note sure why we have a diff on these functions.)

Reply:
They moved up, to be before `bbswap` which now uses them.

Reply:
I don't think `bbswap` relies on them.

Reply:
oh, you are right! it did in an older version, sorry.